### ISSUE TITLE
Deployment Manager support

### ISSUE THREAD

**Issue by Abdennebi** [forseti-security/policy-library#18]
Are you planning to support the Deployment Manager, if yes do you have a schedule otherwise do you need contributions ?

**Comment by morgante** [comment]
Yes, this is definitely something we'd like to support. We're considering either having native platform support or a wrapper similar to Terraform. Do you have a preference?

For the wrapper approach, we'd certainly be open to contributions.

**Comment by Abdennebi** [comment]
Thank you for your quick reply. Ideally, the tool could be called from cloudbuild.yaml (No matter if it native or wrapped).

**Comment by morgante** [comment]
Absolutely, we'd plan for that in either case.
